### PR TITLE
Culture/origin-based language selection

### PR DESCRIPTION
**Pull request by RustingWithYou**
Selectable secondary languages can now be added based on culture/origin rather than just species. Restricts Sol Common to Solarian Skrell as proof of concept.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by KingOfThePing** [comment]
At first I thought "nice", but then I remembered that anyone can learn most languages (as long as there are no biological restrictions). 
So I think this is kind of unncessarily limiting.

**Comment by RustingWithYou** [comment]
> At first I thought "nice", but then I remembered that anyone can learn most languages (as long as there are no biological restrictions). So I think this is kind of unncessarily limiting.

The idea for this is for situations like Skrell speaking Sol Common, where loredevs want it to only be available to a specific background. If I decided to give Ouerean non-Unathi Sinta'Unathi, for instance, this would be a useful mechanic.

**Comment by NonQueueingMatt** [comment]
I've spoken to this with Arrow at length (seriously, we've had a lot of discussions about this PR) and the resolution we both came to is that this is ultimately just another band-aid on a system held together by duct tape. We can't keep adding duct tape to our languages system - it fundamentally does not work and does not make sense because it relies on chains of contorted OOC reasoning and Bay holdovers that make no sense in our universe (tradeband, freespeak) along with unmaintained languages (Sol Common) that should be less monolithic.

In short, this PR is at this time not wanted. Anything that touches languages needs a comprehensive rework by the lore team. I'm willing to put forward my full availability to make sure it happens if there's a concrete effort, and to discuss the head developers' thoughts/visions about it.
